onnxruntime/test/ep_weight_sharing_ctx_gen/main.cc

```diff
@@ -3,24 +3,19 @@
 
 // onnxruntime dependencies
 #include "test_configuration.h"
-#include <core/session/onnxruntime_c_api.h>
 #include <core/session/onnxruntime_cxx_api.h>
-#include <random>
 #include "command_args_parser.h"
 #include <google/protobuf/stubs/common.h>
 
 #include "core/session/onnxruntime_session_options_config_keys.h"
-#include "core/session/inference_session.h"
 #include "core/session/ort_env.h"
```

Comment:
Hi, I believe this is still an internal header. And I also see that this tool still depends on the internal graph classes `onnxruntime::Graph` and `onnxruntime::Node`, which have a public header but a private/internal implementation. Since this still requires the tool to be compiled with internal ORT code, would this prevent users from integrating this into their own toolchains?

Reply:
Good point. Actually for that post processing part, user should be able to use Onnx API to update the Onnx model. That's not the main part we want to cover in this tool.  But anyway, let me make the changes to use Onnx API to make it clear.